packages/components/src/shared/artplayer.ts

```diff
@@ -0,0 +1,71 @@
+import type artplayer from "artplayer/dist/artplayer.js";
+import type { Option as ArtPlayerPluginDanmukuOption } from "artplayer-plugin-danmuku";
+
+export type ArtPlayer = artplayer;
+export type ArtPlayerOptions = typeof artplayer.option;
+
+export interface CustomArtPlayerOptions
+  extends Omit<ArtPlayerOptions, "container" | "url" | "customType"> {
+  container?: HTMLElement | string;
+  url?: string;
+  quality?: {
+    /**
+     * Whether the default is selected
+     */
+    default?: boolean;
+
+    /**
+     * Html string of quality
+     */
+    html: string;
```

Comment:
So what should I fill in?

Reply:
https://artplayer.org/document/start/option.html#quality